File: public/create-icons.js
// Node.js script to create PNG icons
const fs = require('fs');
const { createCanvas } = require('canvas');

function createIcon(size) {
  const canvas = createCanvas(size, size);
  const ctx = canvas.getContext('2d');
  
  // Background gradient
  const gradient = ctx.createLinearGradient(0, 0, size, size);
  gradient.addColorStop(0, '#00d4ff');
  gradient.addColorStop(1, '#667eea');
  ctx.fillStyle = gradient;
  ctx.fillRect(0, 0, size, size);
  
  // Draw brain shape
  ctx.strokeStyle = 'white';
  ctx.lineWidth = size / 32;
  ctx.lineCap = 'round';
  ctx.lineJoin = 'round';
  
  const scale = size / 192;
  
  ctx.save();
  ctx.translate(size/2, size/2);
  ctx.scale(scale, scale);
  
  // Brain outline
  ctx.beginPath();
  ctx.moveTo(-40, 0);
  ctx.bezierCurveTo(-40, -30, -30, -40, 0, -40);
  ctx.bezierCurveTo(30, -40, 40, -30, 40, 0);
  ctx.bezierCurveTo(40, 30, 30, 40, 0, 40);
  ctx.bezierCurveTo(-30, 40, -40, 30, -40, 0);
  ctx.stroke();
  
  // Brain center line
  ctx.beginPath();
  ctx.moveTo(0, -40);
  ctx.lineTo(0, 40);
  ctx.stroke();
  
  // Neural dots
  ctx.fillStyle = 'white';
  const dots = [[-25, -15], [25, -15], [-25, 15], [25, 15], [-15, 0], [15, 0]];
  dots.forEach(([x, y]) => {
    ctx.beginPath();
    ctx.arc(x, y, 3, 0, Math.PI * 2);
    ctx.fill();
  });
  
  ctx.restore();
  
  return canvas.toBuffer('image/png');
}

// Create 192x192 icon
fs.writeFileSync('icon-192.png', createIcon(192));
console.log('Created icon-192.png');

// Create 512x512 icon
fs.writeFileSync('icon-512.png', createIcon(512));
console.log('Created icon-512.png');

File: public/create_icons.py
from PIL import Image, ImageDraw

def create_icon(size):
    # Create a new image with gradient background
    img = Image.new('RGBA', (size, size), (0, 0, 0, 0))
    draw = ImageDraw.Draw(img)
    
    # Create gradient background
    for y in range(size):
        # Gradient from cyan to purple
        r = int(102 * (y / size))
        g = int(126 + (212 - 126) * (1 - y / size))
        b = int(234 + (255 - 234) * (1 - y / size))
        draw.rectangle([(0, y), (size, y+1)], fill=(r, g, b, 255))
    
    # Draw a brain-like circle
    center = size // 2
    radius = size // 3
    draw.ellipse([center - radius, center - radius, center + radius, center + radius], 
                 outline='white', width=size//32)
    
    # Draw center line
    draw.line([(center, center - radius), (center, center + radius)], 
              fill='white', width=size//32)
    
    # Draw horizontal lines for brain texture
    for offset in [-radius//2, 0, radius//2]:
        y = center + offset
        draw.line([(center - radius//2, y), (center + radius//2, y)], 
                  fill='white', width=size//64)
    
    return img

# Create 192x192 icon
icon192 = create_icon(192)
icon192.save('icon-192.png', 'PNG')
print('Created icon-192.png')

# Create 512x512 icon
icon512 = create_icon(512)
icon512.save('icon-512.png', 'PNG')
print('Created icon-512.png')